### PR TITLE
🪟 🎨 Show warning in UI when in workspace that you are not a member of

### LINKED ISSUE #7614: make admin status obvious in the UI
We should display a banner, change the color of the sidebar, have a bright icon in the sidebar, or anything to make it obvious that the user is a superadmin.

### PR DESCRIPTION
**Pull request by joeykmh**
## What
Closes https://github.com/airbytehq/airbyte/issues/7614

Displays a prominent red warning when you are in a cloud workspace that you are not a member of. This is useful when Airbyte employees are in customer workspaces.

https://user-images.githubusercontent.com/7550957/219686516-b49c853a-47bc-46d6-bd45-ed20d24c02b0.mov

This depends on the `FeatureItem.ShowCustomerWorkspaceWarning` feature flag, which is currently off by default. After validating this is desirable to display for all Airbyte staff, we could consider removing the flag.

### CODE REVIEW

File: airbyte-webapp/src/hooks/services/Feature/types.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ export enum FeatureItem {
   AllowOAuthConnector = "ALLOW_OAUTH_CONNECTOR",
   AllowChangeDataGeographies = "ALLOW_CHANGE_DATA_GEOGRAPHIES",
   AllowSyncSubOneHourCronExpressions = "ALLOW_SYNC_SUB_ONE_HOUR_CRON_EXPRESSIONS",
+  ShowCustomerWorkspaceWarning = "SHOW_CUSTOMER_WORKSPACE_WARNING",
```
Comment (timroes) [inline airbyte-webapp/src/hooks/services/Feature/types.tsx:15]: Nitpicky (and sorry saw you already created the LD flag for it): could we maybe rename this to: `SHOW_FOREIGN_WORKSPACE_WARNING` or `SHOW_ADMIN_WARNING`?

File: airbyte-webapp/src/packages/cloud/views/layout/CloudMainView/CloudMainView.tsx [hunk 2]
```diff
@@ -38,6 +39,7 @@ const CloudMainView: React.FC<React.PropsWithChildren<unknown>> = (props) => {
   const workspace = useCurrentWorkspace();
   const cloudWorkspace = useGetCloudWorkspace(workspace.workspaceId);
   const isAllowUpdateConnectorsEnabled = useFeature(FeatureItem.AllowUpdateConnectors);
+  const isShowAdminWarningEnabled = useFeature(FeatureItem.ShowCustomerWorkspaceWarning);
```
Comment (timroes) [inline airbyte-webapp/src/packages/cloud/views/layout/CloudMainView/CloudMainView.tsx:42]: Given we don't set it as a default value, I assume we plan to only enable that for airbyte employees via LD?

File: airbyte-webapp/src/hooks/services/Feature/types.tsx (hunk 1 above)
Comment (josephkmh) [reply airbyte-webapp/src/hooks/services/Feature/types.tsx:15]: Yeah sure, I was just being a bit lazy not renaming these as the UI went through a few iterations 🙂

File: airbyte-webapp/src/packages/cloud/views/layout/CloudMainView/CloudMainView.tsx (hunk 2 above)
Comment (josephkmh) [reply airbyte-webapp/src/packages/cloud/views/layout/CloudMainView/CloudMainView.tsx:42]: Exactly, I think we should do opt-in first, then potentially add it as a default cloud feature if it goes over well

**Comment by josephkmh** [comment]
Superseded by https://github.com/airbytehq/airbyte-platform/pull/124

**Pull request closed without merging**